script/deploy/tron/constants.ts

```diff
@@ -0,0 +1,26 @@
+// Energy and cost constants
+export const ENERGY_PRICE = 0.00021 // TRX per energy unit
```

Comment:
ENERGY_PRICE will always be the same?

In general for this file: 
The comments could be a little more meaningful imho. As this is a new ecosystem, being more explicit helps other developers that didnt do all the research / trial & error that you did and the more knowledge you share/document the better for others.

Like for example in our call you compared some stuff with gas price and gas limit...this could also be added here. 

Min balance for warning.....warning where/when/how? What registration? etc.
Default fee limit for what? the comment just repeats the name basically.

Reply:
Fixed: 08c909e8